legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/resources/ReviewsOnlyResource.java

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.resources;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.finos.legend.sdlc.domain.model.project.ProjectType;
+import org.finos.legend.sdlc.domain.model.review.Review;
+import org.finos.legend.sdlc.domain.model.review.ReviewState;
+import org.finos.legend.sdlc.server.domain.api.review.ReviewApi;
+import org.finos.legend.sdlc.server.time.EndInstant;
+import org.finos.legend.sdlc.server.time.ResolvedInstant;
+import org.finos.legend.sdlc.server.time.StartInstant;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+import java.util.Set;
+
+@Path("/reviews")
+@Api("Reviews")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class ReviewsOnlyResource extends BaseResource
+{
+    private final ReviewApi reviewApi;
+
+    @Inject
+    public ReviewsOnlyResource(ReviewApi reviewApi)
+    {
+        this.reviewApi = reviewApi;
+    }
+
+
+    @GET
+    @ApiOperation(value = "Get reviews without project", notes = "Get reviews for a project. If state is provided, then only reviews with the given state are returned. Otherwise, all reviews are returned. If state is UNKNOWN, results are undefined.")
+    public List<Review> getReviews(@QueryParam("projectTypes") @ApiParam("If not provided or the provided. valid project types would be used") Set<ProjectType> projectTypes,
+                                   @QueryParam("assignedToMe") @ApiParam("show reviews assigned to me, would be set to true is not selected") Boolean  assignedToMe,
```

Comment:
Let's handle the default value at this level.
```suggestion
                                   @QueryParam("assignedToMe")
                                   @DefaultValue("true")
                                   @ApiParam("show reviews assigned to me, would be set to true is not selected") boolean  assignedToMe,
```

Also, shouldn't we have authoredByMe in addition to assignedToMe? I can definitely see a use case for finding all reviews assigned to someone, but I can also see a use case for finding all reviews authored by someone.